### PR TITLE
WIP: Get constituent codes so we can geocode those with Constituent Code "Member"

### PR DESCRIPTION
**Pull request by alexsoble**

### CODE REVIEW

File: jcua_renxt/renxt_client.py [hunk 1]
```diff
@@ -108,6 +107,25 @@ def create_coordinates(self, constituent_id, coordinates_string):
 
         return self.client.request('POST', endpoint, data=request_body_json, headers=headers)
 
+    # GET request.
+    # SKY API docs: https://developer.sky.blackbaud.com/docs/services/56b76470069a0509c8f1c5b3/operations/ListConstituentCodesAllConstituents
+    def get_constituent_codes(self, page_link='https://api.sky.blackbaud.com/constituent/v1/constituents/constituentcodes'):
+        print(f'Page link: {page_link}')
```
Comment (alexsoble) [inline jcua_renxt/renxt_client.py]: Printing all the things for testing purposes to make sure the logic flows as expected. Will remove before asking for review.

File: jcua_renxt/renxt_client.py [hunk 2]
```diff
@@ -108,6 +107,19 @@ def create_coordinates(self, constituent_id, coordinates_string):
 
         return self.client.request('POST', endpoint, data=request_body_json, headers=headers)
 
+    # GET request.
+    # SKY API docs: https://developer.sky.blackbaud.com/docs/services/56b76470069a0509c8f1c5b3/operations/ListConstituentCodesAllConstituents
+    def get_constituent_codes(self, page_link='https://api.sky.blackbaud.com/constituent/v1/constituents/constituentcodes'):
```
[the remark this answers is not in the document]
Comment (emilyfeder) [reply jcua_renxt/renxt_client.py]: What exactly does this endpoint give you? All the constituents with their codes, and then we would filter to only the ones with the code "member" to geocode?

File: jcua_renxt/renxt_client.py [hunk 3]
```diff
@@ -108,6 +107,19 @@ def create_coordinates(self, constituent_id, coordinates_string):
...
+        response = self.client.get(page_link).json()
+        constituent_codes_page = response['value']
+        next_link = response['next_link']
+
+        if len(constituent_codes_page) > 0:
```
[the remark this answers is not in the document]
Comment (emilyfeder) [reply jcua_renxt/renxt_client.py]: Have you tried running this and determining what actually happens?

**Comment by alexsoble** [comment]
Closing this out because @Noleli found a much more efficient endpoint for doing this (we think): https://developer.sky.blackbaud.com/docs/services/56b76470069a0509c8f1c5b3/operations/ListConstituents.

File: jcua_renxt/renxt_client.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexsoble) [reply jcua_renxt/renxt_client.py]: That was the idea . . . @Noleli found what might be a much better endpoint though! https://developer.sky.blackbaud.com/docs/services/56b76470069a0509c8f1c5b3/operations/ListConstituents

**Pull request closed without merging**